### ISSUE TITLE
pav:status on patterns

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#2567]
Can we add a pav:status on patterns? How do we distinguish implemented patterns?

**Comment by matentzn** [comment]
Can you give some more detail? This is a ticket for the DOSDP spec:
https://github.com/INCATools/dead_simple_owl_design_patterns

 But make sure you know what you are asking for.. Just like that I am not sure I understand the point of 
 
 ```
status: implemented
```

That seems to be secondary information that you can see using our `conformsTo` tags on the classes implementing a pattern..

**Comment by nicolevasilevsky** [comment]
Yes, I think this is what Chris wants. This came on the email thread about chromosomal translocations, and I created that new pattern that is just descriptive at the moment, so he wants a way to differentiate those.

Thank you @matentzn!

**Comment by dosumis** [comment]
https://github.com/INCATools/dead_simple_owl_design_patterns/pull/61
